volumes/ecocounter/tables/flows.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE ecocounter.flows (
+    flow_id numeric PRIMARY KEY,
+    site_id numeric NOT NULL REFERENCES ecocounter.sites (site_id),
+    flow_direction text NOT NULL,
+    flow_geom geometry(LINESTRING,4326) CHECK(flow_geom IS NULL OR ST_NPoints(flow_geom) = 2),
+    bin_size interval NOT NULL
+);
+
+GRANT SELECT ON ecocounter.flows TO bdit_humans;
+
+COMMENT ON COLUMN ecocounter.flows.bin_size
+IS 'temporal bins are either 15 or 30 minutes, depending on the sensor';
```

Comment:
is it now all in 15 min bin? Or does the 30 min bin still exist in the old data

Reply:
I'm actually not seeing the half hour bins now, but the hourly bins are still there for sure:

```sql
SELECT *
FROM ecocounter.counts
WHERE flow_id = 101052525
ORDER BY datetime_bin ASC
LIMIT 1000
```